tests/test_bukuDb.py

```diff
@@ -1700,6 +1706,8 @@ def test_sort(bukuDb, fields, ignore_case, expected):
     (False, ['+title', '-tags', 'description', 'index', 'uri'], 'metadata ASC, tags DESC, desc ASC, id ASC, url ASC'),
     (True, ['+title', '-tags', 'description', 'index', 'uri'],
      'LOWER(metadata) ASC, LOWER(tags) DESC, LOWER(desc) ASC, id ASC, LOWER(url) ASC'),
+    (True, ['#foo', '# BaR ', "-# b'A'z ", '# invalid, tag '],
+     "tags LIKE '%,foo,%' ASC, tags LIKE '%,bar,%' ASC, tags LIKE '%,b''a''z,%' DESC")
```

Comment:
Testing SQL ordering generation for tags (tags with comma are invalid and therefore ignored)